payment-instruments/card/seamless-view.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Card Payments – Seamless View
+title: Seamless View
```

Comment:
I've noticed that the Sidebar expects `-` to be present in the title. Does it handle titles without it as well?

Reply:
doest matter because the dx-1195 should be merged in first and the titles are without "-" there